packages/components/package.json

```diff
@@ -23,6 +23,8 @@
     "build": "ember build --environment=production",
     "lint": "npm-run-all --aggregate-output --continue-on-error --parallel \"lint:!(fix)\"",
     "lint:fix": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*:fix",
+    "lint:css": "stylelint \"**/*.scss\"",
```

Comment:
TBD: it seems this runs linting on **all** the files in the `components` folder: not only `app/styles`, but also `tests/dummy/app/styles`. do we need to have two commands for this?

Reply:
I _think_ this is consistent with how eslint is working in the project, but we can confirm.

I would think we'd want to follow the same patterns regardless of where the css is written for consistency sake, but on the other hand most/all of the styles in the test folder will disappear once the website gets moved elsehwere.

Reply:
> most/all of the styles in the test folder will disappear once the website gets moved

yeah, good point. 👍